Symfony/CS/Linter/NullLinter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Linter;
+
+/**
+ * Dummy linter. No linting is performed. No error is raised.
+ *
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ */
+class NullLinter implements LinterInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function lintFile($path)
+    {
+        unset($path);
```

Comment:
wjy unsetting the variable explicitly instead of letting PHP do it at the end of the method ?